src/bio/bio_buffer.c

```diff
@@ -976,8 +986,9 @@ bio_iod_post(struct bio_desc *biod)
 {
 	struct bio_dma_buffer *bdb;
 
-	if (!biod->bd_buffer_prep)
-		return -DER_INVAL;
+	if (!biod->bd_buffer_prep){
```

Comment:
(style)  suspect code indent for conditional statements (8, 24)
(style)  space required before the open brace '{'